bake-kdf/src/lib.rs

```diff
@@ -11,7 +11,7 @@
 use belt_hash::digest::FixedOutput;
 use belt_hash::{BeltHash, Digest, block_api::belt_compress};
 
-/// `belt-keyexpand` key expansion algorithm described in STB 34.101.34-2020 8.1.2.
+/// `belt-keyexpand` key expansion algorithm described in STB 34.101.31-2020 8.1.2.
```

Comment:
The readme references STB 34.101.66-2014 and the linked pdf defines `bake-kdf`, while STB 34.101.31-2020 defines block modes of operation.

Reply:
Ah, and `keyrep` and `keyexpand` should be pub, because it is used in another algorithms, not only in `bake-kdf`.

Reply:
Ah, true. I was confused by the crate name.